R/get_rcode_utils.R

```diff
@@ -46,29 +46,25 @@ get_rcode_str_install <- function() {
 #' @keywords internal
 get_datasets_code <- function(datanames, datasets, hashes) {
   str_prepro <- teal.data:::get_code_dependency(attr(datasets, "preprocessing_code"), names = datanames)
-  if (length(str_prepro) == 0) {
-    str_prepro <- "message('Preprocessing is empty')"
-  } else if (length(str_prepro) > 0) {
-    str_prepro <- paste0(str_prepro, "\n\n")
-  }
+  str_prepro <- ifelse(length(str_prepro) == 0, "message('Preprocessing is empty')", str_prepro)
```

Comment:
This is wrong. `ifelse` always returns a vector the same length as `test`, in this case `1L`. You would lose all but the first element.

`ifelse` is _not_ shorthand for `if (cond) val1 else val2`.